src/main/archetype/pom.xml

```diff
@@ -372,7 +372,7 @@ Bundle-DocURL:
                 <plugin>
                     <groupId>com.day.jcr.vault</groupId>
                     <artifactId>content-package-maven-plugin</artifactId>
-                    <version>1.0.2</version>
+                    <version>1.0.6</version>
```

Comment:
Only version 1.0.4 is available in Maven Central: https://repo1.maven.org/maven2/com/day/jcr/vault/content-package-maven-plugin/

Reply:
Any changelog available?